certifier-osv.md

```diff
@@ -0,0 +1,207 @@
+layout: doc
+permalink: /osv-certifier/
+title: "OSV Certifier Documentation"
+description: "Guide to using the OSV Certifier in GUAC for vulnerability management"
+categories: [certification, vulnerability-management, GUAC]
+tags: [OSV, GUAC, software-supply-chain, security, SBOM]
+---
+
+# OSV Certifier Documentation 
+
+## Overview
+
+The OSV Certifier component of [GUAC](https://guac.sh) (Graph for Understanding Artifact Composition) integrates with the [OSV (Open Source Vulnerability) database](https://osv.dev) to provide vulnerability insights for open-source dependencies. It enables security risk assessment through vulnerability identification in software dependencies.
+
+## Architecture
+
+```
+┌──────────────┐     ┌───────────────┐     ┌────────────┐
+│  SBOM Input  │────>│  OSV Certifier│────>│  OSV API   │
+└──────────────┘     └───────────────┘     └────────────┘
+                            │
+                            ▼
+                    ┌───────────────┐
+                    │  GUAC Graph   │
+                    └───────────────┘
+```
+
+## Key Features
+
+- **Vulnerability Detection**: Scans dependencies using SBOMs and cross-references with OSV's vulnerability catalog
+- **Automated Updates**: Regular synchronization with OSV for current vulnerability data
+- **Comprehensive Reporting**: Structured reports showing vulnerabilities by dependency and severity
+- **SBOM Format Support**: 
```

Comment:
This applies to GUAC generally and not the OSV certifier specifically, so we should probably not include it here. Otherwise, if we add additional SBOM format support, we have many places to update the docs.

Reply:
```suggestion
- **SBOM Format Support**: 
```
Removing that too